### PR TITLE
add plugin for ciw_parser

### PR DESCRIPTION
**Pull request by Fikrisaoudi**
add plugin for ciw_parser with import and export Rspec test

### CODE REVIEW

**Comment by Fikrisaoudi** [comment]
@hammady  @mouradvt04

**Comment by hammady** [comment]
thank you @Fikrisaoudi, I will review the changes. btw, I have installed github/slack plugin, so we will be notified on new commits/issues/PRs so that you don't have to worry about this anymore.

File: lib/rayyan-formats-plugins/plugins/ciw.rb [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+          emit_line("ID", "#{target.keywords.first};"),
+          target.keywords ? target.keywords.map.with_index { |id, i| i+1 == target.keywords.length ? nil : emit_line("  ","#{target.keywords[i+1]};") }: nil,
+          emit_line("LA", target.language),
+          emit_line("UT", target.sid),  
+          "ER\n\nEF"	
+        ].flatten.join if target
+      end
+	 
+      class << self
+	      def get_date_array(article)
```
Comment (hammady) [inline lib/rayyan-formats-plugins/plugins/ciw.rb]: @Fikrisaoudi please fix indentation here

Comment (hammady) [inline lib/rayyan-formats-plugins/plugins/ciw.rb]: and leave spaces around =

File: lib/rayyan-formats-plugins/plugins/ciw.rb [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+              "Journal Article"
+            when 'S'
+              "Conference Proceedings"
+            else
+              publication_types
+            end
+          ]
+        end
+
+	      MONTHS = %w(dummy JAN FEB MAR APR MAY JUN JUL AUG SEP OCT NOV DEC)
```
Comment (hammady) [inline lib/rayyan-formats-plugins/plugins/ciw.rb]: indentation

File: lib/rayyan-formats-plugins/plugins/ciw.rb [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+        def set_convert_date(month)
+         month = MONTHS.index(month) unless month.nil?
```
Comment (hammady) [inline lib/rayyan-formats-plugins/plugins/ciw.rb]: indentation

File: lib/rayyan-formats-plugins/plugins/ciw.rb [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+          .flatten
+          .map {|id|
+            id
+            .split(/[\;]/)
+            .map(&:strip)
+            .reject{|id| id == ""}
+          }.flatten
+        end
+
+	  	  def get_abstrcats(article)
```
Comment (hammady) [inline lib/rayyan-formats-plugins/plugins/ciw.rb]: indentation

File: lib/rayyan-formats-plugins/plugins/ciw.rb [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+          .map{|k| article[k] || []}  # could return arrays or strings
+          .flatten
+          .map{|a| a.split(/#{::RefParsers::NEWLINE_MERGER}\s+/)}
+          .flatten
+        end
+	 
+        def set_publication_type(publication_types)
+          case publication_types
+          when "Journal Article"
+          'J'
```
Comment (hammady) [inline lib/rayyan-formats-plugins/plugins/ciw.rb]: indentation

File: lib/rayyan-formats-plugins/plugins/ciw.rb [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+          when "Conference Proceedings"
+          'S'
+          else
+          publication_types
+          end
+        end
+		
+		    def emit_line(key, value)
```
Comment (hammady) [inline lib/rayyan-formats-plugins/plugins/ciw.rb]: indentation

File: spec/plugins/ciw_spec.rb [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+      end
+    end
+  end
+
+
+  describe ".do_export" do
+    let(:plugin) { CIW }
+    let(:target) {
+      t = RayyanFormats::Target.new
+	    t.publication_types = ['Conference Proceedings']
```
Comment (hammady) [inline spec/plugins/ciw_spec.rb]: indentation

**Pull request merged**